docs/SIL.rst

```diff
@@ -3547,6 +3547,25 @@ by an `end_borrow`_ instruction. All `load_borrow`_ instructions must be
 paired with exactly one `end_borrow`_ instruction along any path through the
 program. Until `end_borrow`_, it is illegal to invalidate or store to ``%0``.
 
+store_borrow
+````````````
+
+::
+
+  sil-instruction ::= 'store_borrow' sil-value 'to' sil-operand
+
+  store_borrow %0 to %1 : $*T
+  // $T must be a loadable type
+  // %1 must be an alloc_stack $T
+
+Stores the value ``%0`` to a stack location ``%1``, which must be an
```

Comment:
Can you add here that this is just the current implementation and the design is not final.

Reply:
done